src/create-json-ad.md

```diff
@@ -0,0 +1,102 @@
+
+At this moment, there are few models created in Atomic Data.
+
+## Setting the first values
+
+```json
+{
+  "https://atomicdata.dev/properties/name": "Writing my first blogpost",
+  "https://atomicdata.dev/properties/description": "Hi! I'm a blogpost. I'm also machine readable!",
+}
+```
+
+## Multiple items
+
+If we want to have _multiple_ items, we can simply use a JSON Array at the root, like so:
+
+```json
+[{
+  "https://atomicdata.dev/properties/name": "Writing my first blogpost",
+  "https://atomicdata.dev/properties/description": "Hi! I'm a blogpost. I'm also machine readable!",
+},{
+  "https://atomicdata.dev/properties/name": "Another blogpost",
+  "https://atomicdata.dev/properties/description": "I'm writing so much my hands hurt.",
+}]
+```
+
+## Preventing deduplication with `localId`
+
+When we want to _publish_ Atomic Data, we also want someone else to be able to _import_ it.
+An important thing to prevent, is _data duplication_.
+If you're importing a list of Blog posts, for example, you'd want to only import every article _once_.
+
+The way to prevent this, is by adding a `localId`.
+This `localId` is used by the importer to find out if it has already imported it before.
+So we, as data producers, need to make sure that our `localId` is _unique_ and _does not change_!
+We can use any type of string that we like, as long as it conforms to these requirements.
+Let's use a unique _slug_, a short name that is often used in URLs.
+
+```json
+{
+  "https://atomicdata.dev/properties/name": "Writing my first blogpost",
+  "https://atomicdata.dev/properties/description": "Hi! I'm a blogpost. I'm also machine readable!",
+  "https://atomicdata.dev/properties/localId": "my-first-blogpost"
+}
+```
+
+## Describing relationships between resources using `localId`
+
+Let's say we also want to describe the `author` of the BlogPost, and give them an e-mail, a profile picture and some biography.
+This means we need to create a new Resource for each Author, and again have to think about the properties relevant for Author.
+We'll also need to create a link from BlogPost to Author, and perhaps the other way around, too.
+
+Normally, when we link things in Atomic Data, we can only use full URLs.
+But, since we don't have URLs yet for our Resources, we'll need a different solution.
+Again, this is where we can use `localId`!
+We can simply refer to the `localId`, instead of some URL that does not exist yet.
+
+```json
+[{
+  "https://atomicdata.dev/properties/name": "Writing my first blogpost",
+  "https://atomicdata.dev/properties/description": "Hi! I'm a blogpost. I'm also machine readable!",
+},{
+  "https://atomicdata.dev/properties/name": "Another blogpost",
+  "https://atomicdata.dev/properties/description": "I'm writing so much my hands hurt.",
+},{
+  "https://atomicdata.dev/properties/firstName": "Johny Author",
+  "https://atomicdata.dev/properties/description": "I'm writing so much my hands hurt.",
```

Comment:
Seems this should be a description of the person, not the blogpost.

Reply:
Edited, thanks!